agreement/pseudonode.go

```diff
@@ -244,21 +270,6 @@ func (n asyncPseudonode) makePseudonodeVerifier(voteVerifier *AsyncVoteVerifier)
 	return pv
 }
 
-// getParticipations retrieves the participation accounts for a given round.
-func (n asyncPseudonode) getParticipations(procName string, round basics.Round) []account.Participation {
```

Comment:
You could hide the caching inside this function instead of requiring the caller to call `loadParticipationKeys`

Reply:
I honestly don't see a huge difference here.. the function effectively was renamed from `getParticipations` -> `loadRoundParticipationKeys`, plus the addition of the caching logic.